packages/cli/src/upload/upload.js

```diff
@@ -342,20 +342,33 @@ async function getPreviousUrlMap(options) {
  * @return {Promise<void>}
  */
 async function writeUrlMapToApi(urlMap) {
+  print(`Saving URL map to temporary public storage...\n`);
```

Comment:
this feels a bit excessive considering we don't even print that much for the github intergration which is much more important. one-line about its status feels enough :)
```suggestion
```